### ISSUE TITLE
Spring Cloud Stream Consumers not detected

### ISSUE THREAD

**Issue by rafaelzig** [springwolf/springwolf-core#684]
**Describe the bug**
In the context of [Spring Cloud Function Support](https://docs.spring.io/spring-cloud-stream/reference/spring-cloud-stream/producing-and-consuming-messages.html#spring_cloud_function), Springwolf does not automatically detect consumer beans declared with `@Component` annotation instead of `@Bean`. 

**Dependencies and versions used**
* springwolf-cloud-stream:1.1.0
* springwolf-ui:1.1.0'

**Code example**
This works:
```java
@Configuration
@Log4j2
public class MyConsumerConfiguration {
  @Bean
  public Consumer<String> myConsumer() {
    return log::info;
  }
}
```

But this does not work:
```java
@Component
@Log4j2
public class MyConsumer implements Consumer<String> {
  @Override
  public void accept(String payload) {
    log.info(payload);
  }
}
```

**Comment by stavshamir** [comment]
Hi @rafaelzig, thanks for reporting this issue. It seems like the cloud stream plugin indeed does not scan `@Component` classes. I'm on it.

**Comment by stavshamir** [comment]
@rafaelzig please try the snapshot version and see if this fix works.

**Comment by rafaelzig** [comment]
@stavshamir Thank you for the swift response.
I can confirm that the issue is resolved with the version `1.2.0-SNAPSHOT`

**Comment by stavshamir** [comment]
@rafaelzig No problem! The fix will be released in 1.3.0 in the beginning of next month.
If you find the project useful, please consider starring it, it helps us spread the word 🙏
